pkg/proc/eval.go

```diff
@@ -496,7 +496,12 @@ func (scope *EvalScope) simpleLocals(flags localsFlags, wantedName string) ([]*V
 		return vars, nil
 	}
 
-	sort.Stable(&variablesByDepthAndDeclLine{vars, depths})
+	sort.SliceStable(vars, func(i, j int) bool {
```

Comment:
This doesn't work because the elements of depths aren't swapped. I made a mistake, I don't think the original can be refactored into sort.SliceStable. It should be left as it is.